core/src/main/scala/kafka/controller/ControllerState.scala

```diff
@@ -104,7 +104,12 @@ object ControllerState {
     def value = 15
   }
 
+  case object UpdateMetadataResponseReceived extends ControllerState {
```

Comment:
Note that this adds a new metric - `UpdateMetadataResponseReceivedRateAndTimeMs`

Reply:
Good point. I'll disable it for now.